R/ensure-lib.R

```diff
@@ -86,6 +90,31 @@ ensure_lib <- function(lib = NULL, test_packages = unlist(strsplit(packageDescri
   lib_dir_path
 }
 
+install_arrow_github <- function(repo = "apache/arrow", ref = "HEAD", ...) {
+  # TODO: check if cmake is found? Say to use brew to install that?
```

Comment:
The build script will download cmake if it isn't found. You may need to revise that script to download a platform/arch appropriate binary though.

Reply:
yeah, the one that is downloaded by default isn't right, but I will poke at that on the arrow side.

Reply:
macos has a different URL: https://cmake.org/download/